newrelic-agent/src/main/java/com/newrelic/agent/instrumentation/ClassNameFilter.java

```diff
@@ -112,15 +124,34 @@ public void addExcludeFileClassFilters() {
                 // ignore
             }
         }
+        boolean ignoreExcludeForSecurityAgent = false;
+        String formattedIgnoredExcludes = String.join(",", SECURITY_AGENT_CLASS_TRANSFORMER_EXCLUDES_TO_IGNORE);
+        boolean shouldInitializeSecurityAgent = shouldInitializeSecurityAgent();
```

Comment:
Only consider removing one of the default exclude rules if the `shouldInitializeSecurityAgent()` check evaluates to `true`.